src/app/(proper_react)/(redesign)/(public)/RebrandAnnouncement.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../tokens";
+
+.rebrandAnnouncement {
+  position: sticky;
+  bottom: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: $spacing-xs;
+  padding: $spacing-lg;
+  background-color: $color-blue-20;
+  font: $text-body-lg;
+
+  @media screen and (max-width: $screen-md) {
+    // (button width) + (.rebrandAnnouncement gap) + (2 * inline button padding)
+    padding-inline-end: calc(24px + calc($spacing-xs + calc(2 * $spacing-sm)));
```

Comment:
Ah! I suppose, I wasn't aware of what the `FalseDoorBanner` looked like. I felt like our banners got designed ad-hoc whenever one is needed, but if they are similar in designs, a general banner component indeed makes sense to me.

As for the specificity, what I wanted to do was add `$spacing-xs` padding in between the button and the text, but since the button is absolutely positioned, I couldn't just add that as a gap - I had to reproduce the button width and insert empty space on the right-hand side of the text. Since I had to calculate what that width would be anyway, I figured I'd might as well make it explicit in code to avoid a magic number and to make it easier to re-calculate if the button width is ever adjusted.